int32 lifter_control_kind
